utils/src/main/java/com/revenuecat/purchases/VerificationResult.kt

```diff
@@ -26,5 +26,10 @@ enum class VerificationResult {
     /**
      * Verification failed, possibly due to a MiTM attack.
      */
-    FAILED
+    FAILED,
+
+    /**
+     * Entitlements were computed on device.
+     */
+    ENTITLEMENTS_COMPUTED_ON_DEVICE
```

Comment:
I had some doubts about the correct preposition to use here (on vs in). So I asked ChatGTP, it's clear now :P 
<img width="673" alt="image" src="https://user-images.githubusercontent.com/808417/229830116-3aa54b74-c4c8-434a-bdbc-20deb8fc2873.png">